src/scss/_vars.scss

```diff
@@ -13,6 +13,7 @@ $white: #fff;
 // Secondary color palette
 $soft-black: #545454;
 $dark-gray: #7b7474;
+$keyline-gray: #8f8989;
```

Comment:
your color is being overwritten by line 19. This color token already exist. If the figma is referencing another color value we should call that out in the ticket but not modify this token until verifying that all uses of keyline-gray are now changing.

Reply:
Oh shoot, I missed that, I did a search for the hex code, not the name. This is the same problem I had with sale lot redesign having two Soft-Black's defined in figma. Who should I reach out to to confirm?